lib/tasks/content_tagged_to_ancestors.rake

```diff
@@ -0,0 +1,11 @@
+namespace :content do
+  desc "Find all content that is tagged to a taxon and its direct ancestor and removes the redundant tagging"
+  task content_tagged_to_ancestor: :environment do
+    results = Tagging::CommonAncestorFinder.call
+    results.each do |result|
+      puts "Document #{result[:title]}, content_id: #{result[:content_id]} has common ancestors tagged to: "
+      puts result[:common_ancestors].inspect
+      Tagging::Untagger.call(result[:content_id], result[:common_ancestors])
```

Comment:
It might be useful to have a way of running this, without it changing everything, just so we can manually check some of the things it would do before it does them. What do you think?